bigtable/src/filter_composing_chain.php

```diff
@@ -48,20 +48,17 @@ function filter_composing_chain(
     $filter = Filter::chain()
         ->addFilter(Filter::limit()->cellsPerColumn(1))
         ->addFilter(Filter::family()->exactMatch("cell_plan"));
-    read_filter($table, $filter);
-}
-
-// Helper function for printing the filtered data
-function read_filter($table, $filter)
-{
+    
     $rows = $table->readRows([
         'filter' => $filter
     ]);
 
     foreach ($rows as $key => $row) {
+        // helper function defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```

Comment:
Nit, but let's use a complete sentence here to be absolutely crystal clear:
```suggestion
        // The "print_row" helper function is defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```